extension/src/experiments/columns/model.test.ts

```diff
@@ -9,30 +9,62 @@ import outputFixture from '../../test/fixtures/expShow/output'
 import columnsFixture from '../../test/fixtures/expShow/columns'
 import {
   deeplyNestedOutput,
-  columns as deeplyNestedColumns
+  columns as deeplyNestedColumns,
+  columnsWithDepthOf10 as deeplyNestedColumnsWithDepthOf10,
+  columnsWithDepthOf3 as deeplyNestedColumnsWithDepthOf3
 } from '../../test/fixtures/expShow/deeplyNested'
 import {
   dataTypesOutput,
   columns as dataTypesColumns
 } from '../../test/fixtures/expShow/dataTypes'
+import { getConfigValue } from '../../vscode/config'
 
 jest.mock('../../vscode/config')
 
+const mockedGetConfigValue = jest.mocked(getConfigValue)
+mockedGetConfigValue.mockImplementation(() => 5)
```

Comment:
[I] Can also use a `beforeEach` this time I would but the `mockReturnValueOnce` calls inside of the tests as the value is being varied throughout the file